### PR TITLE
[JBMETA-440]: Use urn instead of URL for jboss namespaces.

### PR DESCRIPTION
**Pull request by ehsavoie**

### CODE REVIEW

**Review by bstansberry — APPROVED** [review]

File: appclient/src/main/resources/schema/jboss-client_9_0.xsd [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="urn:jboss:jakartaee:1.0
+                            https://www.jboss.org/j2ee/schema/jboss-client_9_0.xsd"
+             version="9.0">
+	       ...
+            </jboss-client>
+
+	The instance documents may indicate the published version of
+	the schema using the xsi:schemaLocation attribute for the
+	Java EE namespace with the following location:
```
Comment (jbliznak) [inline appclient/src/main/resources/schema/jboss-client_9_0.xsd:51]: Java EE -> Jakarta EE ?

File: appclient/src/main/resources/schema/jboss-client_9_0.xsd [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+	urn:jboss:jakartaee:1.0
+
+	and by indicating the version of the schema by
+	using the version attribute as shown below:
+
+	    <jboss-client xmlns="urn:jboss:jakartaee:1.0"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="urn:jboss:jakartaee:1.0
+                            https://www.jboss.org/j2ee/schema/jboss-client_9_0.xsd"
```
Comment (jbliznak) [inline appclient/src/main/resources/schema/jboss-client_9_0.xsd:44]: different location then on line 53
https://www.jboss.org/j2ee/schema/jboss-client_9_0.xsd
vs
https://www.jboss.org/schema/jbossas/jboss-client_9_0.xsd

File: appclient/src/main/resources/schema/jboss-client_9_0.xsd [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            targetNamespace="urn:jboss:jakartaee:1.0"
+            version="9.0"
+            elementFormDefault="qualified"
+        >
+
+   <xsd:annotation>
+      <xsd:documentation>
+         <![CDATA[
+
+	This is the XML Schema for the JBoss 8.0 application client deployment descriptor.
```
Comment (jbliznak) [inline appclient/src/main/resources/schema/jboss-client_9_0.xsd:31]: 8.0 ?

File: ear/src/main/resources/schema/jboss-app_9_0.xsd [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+           xmlns:jboss="urn:jboss:jakartaee:1.0"
+           targetNamespace="urn:jboss:jakartaee:1.0"
+           version="9.0"
+           elementFormDefault="qualified">
+
+    <xs:annotation>
+        <xs:documentation>
+            <![CDATA[
+
+	This is the XML Schema for the JBoss AS7 EAR application deployment descriptor.
```
Comment (jbliznak) [inline ear/src/main/resources/schema/jboss-app_9_0.xsd:31]: JBoss AS7 seems obsolete

File: ear/src/main/resources/schema/jboss-app_9_0.xsd [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+	The deployment descriptor must be named "jboss-app.xml" and placed in the
+	.ear/META-INF folder. All the descriptors must indicate
+	the JBoss schema by using the Java EE namespace:
```
Comment (jbliznak) [inline ear/src/main/resources/schema/jboss-app_9_0.xsd:34]: Java EE -> Jakarta EE ?

File: ear/src/main/resources/schema/jboss-app_9_0.xsd [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+	    <jboss-app xmlns="urn:jboss:jakartaee:1.0"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             version="9.0">
+	       ...
+            </jboss-app>
+
+    The instance documents may indicate the published version of
+	the schema using the xsi:schemaLocation attribute for the
+	Java EE namespace with the following location:
```
Comment (jbliznak) [inline ear/src/main/resources/schema/jboss-app_9_0.xsd:49]: Java EE -> Jakarta EE ?

File: ear/src/main/resources/schema/jboss-app_9_0.xsd [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+	 https://www.jboss.org/schema/jbossas/jboss-app_10_0.xsd
```
Comment (jbliznak) [inline ear/src/main/resources/schema/jboss-app_9_0.xsd:51]: version mismatch 9 vs 10 ?

File: appclient/src/main/resources/schema/jboss-client_9_0.xsd [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+	the schema using the xsi:schemaLocation attribute for the
+	Java EE namespace with the following location:
+
+	https://www.jboss.org/schema/jbossas/jboss-client_9_0.xsd
+
+	]]>
+      </xsd:documentation>
+   </xsd:annotation>
+
+   <xsd:import namespace="https://jakarta.ee/xml/ns/jakartaee" schemaLocation="https://jakarta.ee/xml/ns/jakartaee/application-client_10.xsd"/>
```
Comment (jbliznak) [inline appclient/src/main/resources/schema/jboss-client_9_0.xsd:59]: Why is it ok to have JBoss schemas versioned as 9 while they builds up on Jakarta EE10?

Comment (ehsavoie) [reply appclient/src/main/resources/schema/jboss-client_9_0.xsd:59]: Because they don't relate

**Pull request merged**